x-pack/plugins/apm/server/lib/services/get_services/get_services_items_stats.ts

```diff
@@ -51,6 +70,16 @@ export const getTransactionDurationAverages = async ({
                   field: TRANSACTION_DURATION,
                 },
               },
+              over_time: {
```

Comment:
Instead of `over_time`, should it be called `timeseries`, or perhaps just `series`? 
I didn't understand what `over_time` meant until I saw the implementation

Reply:
Sure, agree that `timeseries` is more clear.